src/code-generators/typescript-generators/namedImport.ts

```diff
@@ -53,26 +57,54 @@ export function generateNamedImport(
     const lib = `${stringQuoteStyle}${imp.libraryName}${stringQuoteStyle}${eol}`;
 
     const specifiers = imp.specifiers.sort(specifierSort).map(o => generateSymbolSpecifier(o)).join(', ');
-    let importSpecifiers = `${space}${specifiers}${space}`;
-    if (importSpecifiers.trim().length === 0) {
-        importSpecifiers = ' ';
-    }
-
-    const importString = importTemplate(
-        getImportSpecifiers(imp, spaceBraces),
-        lib,
-    );
+    let retVal:string = '';
 
-    if (importString.length > multiLineWrapThreshold) {
+    if (specifiers.length > multiLineWrapThreshold) {
         const spacings = Array(tabSize + 1).join(' ');
-        return multiLineImport(
-            imp.specifiers.sort(specifierSort).map(o => `${spacings}${generateSymbolSpecifier(o)}`).join(',\n'),
-            multiLineTrailingComma ? ',' : '',
-            `${stringQuoteStyle}${imp.libraryName}${stringQuoteStyle}${eol}`,
-            imp.defaultAlias ? `${imp.defaultAlias}, ` : '',
+        const sortedImportSpecifiers: SymbolSpecifier[] = imp.specifiers.sort(specifierSort);
+        let importSpecifierStrings: string = '';
+
+        if (multiLineWrapMethod === 'MULTIPLE_IMPORTS_PER_LINE') {
```

Comment:
Please use the string enum, as mentioned above

Reply:
yes.